go.sum

```diff
@@ -601,6 +605,8 @@ github.com/rogpeppe/go-internal v1.12.0/go.mod h1:E+RYuTGaKKdloAfM02xzb0FW3Paa99
 github.com/russross/blackfriday v1.6.0 h1:KqfZb0pUVN2lYqZUYRddxF4OR8ZMURnJIG5Y3VRLtww=
 github.com/russross/blackfriday v1.6.0/go.mod h1:ti0ldHuxg49ri4ksnFxlkCfN+hvslNlmVHqNRXXJNAY=
 github.com/russross/blackfriday/v2 v2.1.0/go.mod h1:+Rmxgy9KzJVeS9/2gXHxylqXiyQDYRxCVz55jmeOWTM=
+github.com/sagikazarmark/locafero v0.6.0 h1:ON7AQg37yzcRPU69mt7gwhFEBwxI6P9T4Qu3N51bwOk=
```

Comment:
Is this a library we're already using?

Reply:
I added [viper](https://github.com/spf13/viper) as a dependency to help managing the different envioronment variables as a config struct ( see [ClusterConfig](https://github.com/Azure/ARO-RP/blob/db9f2241faf333ec890ecc19dc8fd57d2819fb56/pkg/util/cluster/cluster.go#L57) definition and viper usage in [NewClusterConfigFromEnv](https://github.com/Azure/ARO-RP/blob/db9f2241faf333ec890ecc19dc8fd57d2819fb56/pkg/util/cluster/cluster.go#L121))

All the addiitonal dependencies are pulled in as dependencies of viper itself.

Reply:
It was done to fix this e2e failure: https://issues.redhat.com/browse/ARO-14420
Reasoning is in comments.